x-pack/plugins/beats_management/public/components/table/action_schema.ts

```diff
@@ -83,17 +83,17 @@ export const tagConfigActions: ControlSchema[] = [
     danger: true,
     grow: false,
     name: i18n.translate('xpack.beatsManagement.tagConfigAssignmentOptions.removeTagsButtonLabel', {
-      defaultMessage: 'Remove tag(s)',
+      defaultMessage: 'Remove tag',
     }),
     type: ActionComponentType.Action,
     showWarning: true,
     warningHeading: i18n.translate(
       'xpack.beatsManagement.tagConfigAssignmentOptions.removeTagsWarninigTitle',
-      { defaultMessage: 'Remove tag(s)' }
+      { defaultMessage: 'Remove tag?' }
```

Comment:
@mattapperson Same comment as above. The text should be singular because it's relevant when users try to remove a tag assigned to one or more Beats. Here's what the dialog looks like:

![image](https://user-images.githubusercontent.com/14206422/54163935-ec3f0380-4417-11e9-92b8-18bde0912e6a.png)

Reply:
Even when the user has more then one selected? Maybe we should use 2 labels?

Reply:
@mattapperson Is this dialog accessible from somewhere other than the Update tag page? (shown in the screen capture [here](https://github.com/elastic/kibana/pull/30338#discussion_r264465294))?  If it is, I would suggest that you use a different dialog rather than sharing it. The problem is that "Remove tags" in this context might lead users to believe that all tags are removed (since they haven't selected anything). Make sense?

Reply:
👍🏻